### ISSUE TITLE
Programme working???

### ISSUE THREAD

**Issue by Tritovator** [rcladmin/cordova-sample-contacts#10]
Just checking to see if this was the final working project.

**Comment by rcladmin** [comment]
Yes the app is working, the plugins and platform folders are not included in the repository. The Readme will give you instructions on how to deploy the app.
